src/behat/Api/Context/RestContextTrait.php

```diff
@@ -86,6 +83,30 @@ public function locatePath($path)
      */
     abstract protected function getBaseUri();
 
+    /**
+     * Parse URI path
+     *
+     * @param string $path
+     * @return string
+     */
+    public function locatePath($path)
+    {
+        return 0 !== strpos($path, 'http')
+            ? rtrim($this->getBaseUri(), '/') . '/' . ltrim($path, '/')
+            : $path;
+    }
+
+    /**
+     * Initialize validator according to centreon web api documentation
+     *
+     * @Given the endpoints are described in Centreon Web API documentation
+     */
+    public function theCentreonApiDocumentation()
+    {
+        $this->apiValidator = (new ValidatorBuilder())
+            ->fromYamlFile(__DIR__ . '/../../../../../../../doc/API/centreon-api-v2.yaml');
```

Comment:
Caution! The version number may change at a later date.

Reply:
As we discussed, api version management will be updated.
IMHO, on-premise solution don't need to manage retro-compatibility on major version.
So api yaml file will be linked to a git branch, not in separated files